playbooks/matrix-mult.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Run GPU Matrix Multiply to validate the GPU deployment
+- name: Run GPU Matrix Mult
```

Comment:
no need for the comment, name can be `Run mlperf (or whatever) Matrix Multiplication on the GPU`